### PR TITLE
New package: python-vispy-0.4.0

### PR DESCRIPTION
**Pull request by jnbr**

### CODE REVIEW

File: srcpkgs/python-vispy/template [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+depends="python-numpy"
+short_desc="Python library for interactive scientific visualization"
+maintainer="johannes <johannes.brechtmann@gmail.com>"
+license="BSD"
+homepage="http://vispy.org"
+distfiles="${PYPI_SITE}/v/vispy/vispy-${version}.tar.gz"
+checksum=6bc03ec02825b57645cdf67fe04cd6c287bbd96d41d025a2c008d6bdd210d63f
+
+post_install() {
+	vlicense "${FILESDIR}/LICENSE.txt"
```
Comment (asergi) [inline srcpkgs/python-vispy/template]: `$XBPS_FETCH_CMD https://raw.githubusercontent.com/vispy/vispy/master/LICENSE.txt`
`vlicense LICENSE.txt`

File: srcpkgs/python-vispy/template [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+pkgname=python-vispy
+version=0.4.0
+revision=1
+noarch=yes
+pycompile_module="vispy"
+build_style=python-module
+wrksrc="vispy-${version}"
+hostmakedepends="python-setuptools python3-setuptools"
+depends="python-numpy"
+short_desc="Python library for interactive scientific visualization"
```
Comment (asergi) [inline srcpkgs/python-vispy/template]: For the sake of consistency: `short_desc="Python2 library for interactive scientific visualization"`

File: srcpkgs/python-vispy/template [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+homepage="http://vispy.org"
+distfiles="${PYPI_SITE}/v/vispy/vispy-${version}.tar.gz"
+checksum=6bc03ec02825b57645cdf67fe04cd6c287bbd96d41d025a2c008d6bdd210d63f
+
+post_install() {
+	vlicense "${FILESDIR}/LICENSE.txt"
+}
+
+python3-vispy_package() {
+	short_desc="${short_desc/Python/Python3}"
```
Comment (asergi) [inline srcpkgs/python-vispy/template]: Ditto: `short_desc="${short_desc/Python2/Python3}"`

**Comment by jnbr** [comment]
changed `short_desc` for Python2 and removed `LICENSE.txt` from files, fetching it in `post_install` instead

**Pull request merged**